src/content/configuration/output.md

```diff
@@ -750,6 +750,47 @@ module.exports = {
 };
 ```
 
+`libraryTarget: 'system'` - This will expose your library as a [`System.register`](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md)
+module. This feature was first released in [webpack@4.30.0](https://github.com/webpack/webpack/releases/tag/v4.30.0).
```

Comment:
Sorry for missing this from the first time but preferred way to mention a version is `webpack 4.30.0`, you can find the relevant info here: https://webpack.js.org/contribute/writers-guide/#typesetting

```suggestion
module. This feature was first released in [webpack 4.30.0](https://github.com/webpack/webpack/releases/tag/v4.30.0).
```